exporters/CMakeLists.txt

```diff
@@ -1,3 +1,17 @@
+# Copyright 2021, OpenTelemetry Authors
```

Comment:
Somehow I haven't noticed this before...
Do we need to include the actual year, or we can do something simpler like https://github.com/open-telemetry/opentelemetry-dotnet/blob/381908c089aac4fa9a2056d6bbf465b9853b81b8/src/OpenTelemetry/AssemblyInfo.cs#L2?

Reply:
Not sure where I copied from. Have fixed it now. Thanks for the comment.